### PR TITLE
feat(config): add form field

### PR DESCRIPTION
**Pull request by wanlce**
- pprof_port: 0
- enable_local_tcp_fast_redirect: false
- mptcp: false
- bandwidth_max_tx: 200 mbps
- bandwidth_max_rx: 1 gbps

### Checklist

- [x] The Pull Request has been fully tested
- [ ] There's an entry in the CHANGELOGS
- [ ] There is a user-facing docs PR against https://github.com/daeuniverse/daed

### Full Changelogs

- [Implement ...]

### Issue Reference

Closes #_[issue number]_

### CODE REVIEW

File: src/i18n/locales/zh-Hans.json [hunk 1]
```diff
@@ -108,7 +108,11 @@
       "tproxyPortProtect": "将其设置为 true 可保护透明代理端口免受未经请求的流量的影响。将其设置为 false 以允许用户使用自我管理的 iptables 透明代理规则。",
       "udpCheckDns": "此 DNS 将用于检查节点的 UDP 连接。如果下面的 DNS 上游包含 TCP，它也可以用于检查节点的 TCP DNS 连接。第一个是 URL，其他是 IP地址（如果给定的话）。如果您在本地有 dual stack（双协议栈），则此 DNS 应该同时具有 IPv4 和 IPv6。",
       "utlsImitate": "要模仿的 uTLS 的客户端 Hello ID。只有当 TLS 实现 为 utls 时，此操作才会生效。",
-      "wanInterface": "要绑定的 WAN 接口。如果您想代理本机，请使用它。"
+      "wanInterface": "要绑定的 WAN 接口。如果您想代理本机，请使用它。",
+      "pprofPort": "pprof 端口",
+      "mptcp": "启用 MPTCP。",
+      "bandwidthMaxRx": "最大接收带宽。'200 mbps' # uplink, or '200 m' or '200 mb' or '200 mbps' or 25000000 (which is 200/8*1000*1000)",
+      "bandwidthMaxTx": "最大发送带宽。'1 gbps' # downlink, or '1 g' or '1 gb' or '1 gbps' or 125000000 (which is 1000/8*1000*1000)"
```
Comment (kunish) [inline src/i18n/locales/zh-Hans.json:115]: maybe we should do translation on these comments as well?

File: src/i18n/locales/zh-Hans.json [hunk 2]
```diff
@@ -174,6 +178,11 @@
   "tlsImplementation": "TLS 实现",
   "tproxyPort": "透明代理端口",
   "tproxyPortProtect": "透明代理端口保护",
+  "pprofPort": "pprof 端口",
+  "enableLocalTcpFastRedirect": "启用本地 TCP 快速重定向",
+  "mptcp": "MPTCP",
+  "bandwidthMaxTx": "最大发送带宽",
+  "bandwidthMaxRx": "最大接收带宽",
```
Comment (kunish) [inline src/i18n/locales/zh-Hans.json:185]: And add those texts above

File: src/components/ConfigFormModal.tsx [hunk 3]
```diff
@@ -296,6 +311,15 @@ export const ConfigFormDrawer = forwardRef(({ opened, onClose }: { opened: boole
                     })}
                   />
 
+                  <NumberInput
+                    label={t('pprofPort')}
```
Comment (kunish) [inline src/components/ConfigFormModal.tsx:315]: Is this required field from a user's perspective?

File: src/components/ConfigFormModal.tsx [hunk 4]
```diff
@@ -483,6 +521,18 @@ export const ConfigFormDrawer = forwardRef(({ opened, onClose }: { opened: boole
                       {...form.getInputProps('utlsImitate')}
                     />
                   )}
+
+                  <TextInput
+                    label={`${t('bandwidthMaxTx')}`}
```
Comment (kunish) [inline src/components/ConfigFormModal.tsx:526]: `` is not necessary

File: src/components/ConfigFormModal.tsx (hunk 3 above)
Comment (wanlce) [reply src/components/ConfigFormModal.tsx:315]: pprofPort doesn't seem to work.

Comment (kunish) [reply src/components/ConfigFormModal.tsx:315]: > pprofPort doesn't seem to work.

It should not be marked as withAsterisk then

File: src/components/ConfigFormModal.tsx (hunk 4 above)
Comment (wanlce) [reply src/components/ConfigFormModal.tsx]: Done

File: src/i18n/locales/zh-Hans.json (hunk 2 above)
Comment (wanlce) [reply src/i18n/locales/zh-Hans.json]: Done

File: src/components/ConfigFormModal.tsx (hunk 3 above)
Comment (wanlce) [reply src/components/ConfigFormModal.tsx]: I set up pprofPort on daed, but the corresponding port didn't start. I don't know why this happened, so I can only temporarily hide it from the interface.

**Review by kunish — APPROVED** [review]
LGTM bro :)

**Review by MarksonHon — APPROVED** [review]

**Pull request merged**